pylint/lint/report_functions.py

```diff
@@ -27,23 +30,25 @@ def report_messages_stats(sect, stats, _):
         if not msg_id.startswith("I")
     )
     in_order.reverse()
-    lines = ("message id", "occurrences")
+    lines: Tuple[str, str] = ("message id", "occurrences")
     for value, msg_id in in_order:
-        lines += (msg_id, str(value))
-    sect.append(report_nodes.Table(children=lines, cols=2, rheaders=1))
+        lines += (msg_id, str(value))  # type: ignore
```

Comment:
`type: ignore` not needed if it's a list.

Reply:
```suggestion
        lines += (msg_id, str(value))
```